serialization-protobuf/src/main/kotlin/io/github/kkafka/consumer/flowProtobuf.kt

```diff
@@ -0,0 +1,21 @@
+package io.github.kkafka.consumer
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import kotlinx.serialization.DeserializationStrategy
+import kotlinx.serialization.protobuf.ProtoBuf
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import java.time.Duration
+
+public fun <K, V> KafkaConsumer<ByteArray, ByteArray>.pollWithFlow(
+    pollingPeriod: Duration,
+    keyDeserializer: DeserializationStrategy<K>,
+    valueDeserializer: DeserializationStrategy<V>,
+    protobuf: ProtoBuf = ProtoBuf
+): Flow<ConsumerRecord<K, V>> = pollWithFlow(pollingPeriod)
+    .map {
```

Comment:
What's the difference between `map` used here and `onEach` used in the generic flow example?

Reply:
map(lamnda: A -> B) returns a flow of B from a flow of A

onEach(lambda: A -> Unit) returns a flow of the same type where lambda is called on each element upon collection of the flow (aka lazily)

same semantics as a Sequence or a java Stream